starboard/nplb/socket_clear_last_error_test.cc

```diff
@@ -14,7 +14,7 @@
 
 // The Sunny Day test is used in several other tests as the way to detect if the
 // error is correct, so it is not repeated here.
-
+#if SB_API_VERSION < 17
```

Comment:
Delete these sections instead of guarding them with < 17?

Reply:
This is the same thing as the previous socket header cleanup -- I think we will need to re-write these tests, but I want to get the header clean up in and will address the tests in a follow up since I'm not sure which tests are still relevant and need to be posixified, or just deleted.